src/ops/devToArticles.js

```diff
@@ -0,0 +1,15 @@
+const fetch = require("node-fetch");
+
+module.exports = async (newsChannel) => {
+  const resp = await fetch("https://dev.to/api/articles");
+  const json = await resp.json();
+
+  newsChannel.send(
```

Comment:
Wouldn't you want to `await` or `Promise.all()` this in case it takes a minute to return `resp` or `json`?

Reply:
I dont understand.  Line 5 blocks the process